mon-pix/app/services/session.js

```diff
@@ -92,39 +92,25 @@ export default class CurrentSessionService extends SessionService {
     const domain = this.currentDomain.getExtension();
 
     if (domain === FRANCE_TLD) {
-      await this.locale.setLocale(DEFAULT_LOCALE);
+      this.locale.setLocale(DEFAULT_LOCALE);
```

Comment:
cf. https://github.com/1024pix/pix/blob/dev/orga/app/services/session.js et https://github.com/1024pix/pix/blob/dev/certif/app/services/session.js j'aurais mis plutôt `FRENCH_INTERNATIONAL_LOCALE` que `DEFAULT_LOCALE`, d'autant que c'est la constante `FRENCH_INTERNATIONAL_LOCALE` qui est utilisée dans le test correspondant.


```suggestion
      this.locale.setLocale(DEFAULT_LOCALE);
```

Reply:
Je comprends que c'est pour des raisons de "lisibilité".
Le seul point que je peux soulever de faire ce choix, c'est que la constante `DEFAULT_LOCALE` devient obsolète et on devrait la retirer complètement si on souhaite être plus explicite dans le nommage.

Reply:
Je pense qu'il serait bon dans cette PR faire la modification `this.locale.setLocale(DEFAULT_LOCALE)` →       `this.locale.setLocale(FRENCH_INTERNATIONAL_LOCALE)`, car c'est plus lisible, plus logique et synchro avec les autres apps. Et puis nous pourrons discuter plus tard de la pertinence de garder `DEFAULT_LOCALE`. Effectivement la question se pose. Je suis pour garder encore la constante `DEFAULT_LOCALE` au cas où nous en aurions besoin dans les dernières PR sur le sujet, et s'il s'avère qu'elle n'est vraiment pas nécessaire nous le verrons clairement 😁